src/comm/LinkManager.cc

```diff
@@ -481,7 +481,10 @@ void LinkManager::_updateConfigurationList(void)
                 }
             } else {
                 // Lets create a new Serial configuration automatically
-                pSerial = new SerialConfiguration(QString("Pixhawk on %1").arg(portInfo.portName().trimmed()));
+		if (portInfo.description() == "AeroCore")
```

Comment:
> Can we get other boards that could come across with vendor id = 9900 but not be a Pixhawk? Wondering if there should be a description string check for Pixhawk (or other things) here on the other side of the if?

Good question. The vendor id 9900 (0x26AC) doesn't seem to be a real, registered id. At least it's not in the official database. Provided all PX4 controllers out there return the same *description*, we could certainly use the same logic used here to identify the AeroCore.